pandas/plotting/_core.py

```diff
@@ -308,10 +308,21 @@ def _setup_subplots(self):
 
         axes = _flatten(axes)
 
-        if self.logx or self.loglog:
+        valid_log = {False, True, 'sym', None}
+        input_log = {self.logx, self.logy, self.loglog}
+        if input_log - valid_log:
+            raise ValueError(f"Valid inputs are boolean, None and 'sym'"
```

Comment:
We can't use f-strings yet, unfortunately.

Reply:
sorry, changed! thanks! @TomAugspurger

Reply:
May need to push these changes.

Reply:
ahh, i am really really sorry... i thought i successfully pushed the change before rushing home @TomAugspurger